src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -0,0 +1,99 @@
+.profile-container {
+  font-size: 0.9em;
+  width: 1200px;
+  display: flex;
+  justify-content: center;
```

Comment:
These lines are repeating 4 times in this scss file. You can do @mixin for them.
But it's optional, as you want.